consumption.tex

```diff
@@ -2,57 +2,44 @@ \section{Consumption}
 
 Over the last 5 years there has been an explosion of innovation through new cryptocurrencies. Unfortunately, it is very hard for a new cryptocurrency to gain traction. Without traction, no market depth ensues and a cryptocurrency has difficulty getting listed in any exchange. But without being listed in any exchange, a cryptocurrency cannot gain sufficient traction.
 
-This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, for it to gain traction. We propose utilizing proof-of-burn to allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy cryptocurrency.
-
-We wish to allow users to exchange a legacy cryptocurrency for a new cryptocurrency. We call the former the \emph{source cryptocurrency}, and its blockchain the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
-
-We now describe how a smart contract on the target blockchain can verify that a burn has taken place on the source blockchain. Note how this puts a constraint on the target blockchain to have smart contract capabilities. However there is no such constraint for the source blockchain. Such a mechanism allows a target cryptocurrency to bootstrap through burns on another source blockchain as outlined in Section~\ref{section:introduction}.
+This chicken-and-egg situation presents the need for a solution that circumvents exchanges and allows users to acquire the cryptocurrency in a usable manner, for it to gain traction. We propose utilizing proof-of-burn to allow users to obtain capital on a new cryptocurrency, automatically and without trusted third parties, by burning a legacy cryptocurrency. We call the former the \emph{source cryptocurrency}, and its blockchain the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}. The target blockchain may support burning from multiple source blockchains.
```

Comment:
" in a usable manner, for it to gain traction" -> "directly"

Reply:
rm ", automatically and without trusted third parties", we talk about this later

Reply:
> We call the former the \emph{source cryptocurrency}, and its blockchain the \emph{source blockchain}. We call the latter the \emph{target cryptocurrency} and its corresponding blockchain the \emph{target blockchain}.

->

> We call them the \emph{source} and \emph{target cryptocurrency}, and their blockchains the \emph{source} and {target blockchain} respectively.